server/command.go

```diff
@@ -208,14 +208,16 @@ func (p *Plugin) ExecuteCommand(_ *plugin.Context, args *model.CommandArgs) (*mo
 		case SettingNotifications:
 			if value {
 				if err := p.storeGitlabToUserIDMapping(info.GitlabUsername, info.UserID); err != nil {
-					p.API.LogError("can't store GitLab user id mapping", "err", err.Error())
+					p.API.LogError("can't store GitLab to user id mapping", "err", err.Error())
 					return p.getCommandResponse(args, "Unknown error please retry or ask to an administrator to look at logs"), nil
 				}
-			} else {
-				if err := p.API.KVDelete(info.GitlabUsername + GitlabUsernameKey); err != nil {
-					p.API.LogError("can't delete GitLab username in kvstore", "err", err.Error())
+				if err := p.storeGitlabIDToUserIDMapping(info.GitlabUsername, info.GitlabUserID); err != nil {
+					p.API.LogError("can't store GitLab to GitLab id mapping", "err", err.Error())
 					return p.getCommandResponse(args, "Unknown error please retry or ask to an administrator to look at logs"), nil
 				}
+			} else if err := p.API.KVDelete(info.GitlabUsername + GitlabUsernameKey); err != nil {
```

Comment:
2/5 wrap this into a `p.deleteXXX` function so it's more intention-revealing and consistent with `store...`

Reply:
Sure, thanks :)

Reply:
Done, I also refactored two other similar functions so that it´s more consistent :v: